plugins/utilities.json

```diff
@@ -3,6 +3,25 @@
   "description": "Utilities",
   "plugins_base_url": "http://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "mood_light": {
+      "description": "mood lighting in co-op games",
```

Comment:
"mood lighting" seems unclear, let's elaborate it

Reply:
I really don't know how to eloborate it

Reply:
how about "Dynamic lighting in co-op games"? let's also mention about `cm` chat command for changing plugin settings while in-game